dev-libs/sway/sway-9999.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
@nvinson, I object. I'm  looking forward to this getting merged, and I've been using live for a while.